### ISSUE TITLE
Clicking on rule does not center view in 2.18.0

### ISSUE THREAD

**Issue by kevfengler227** [igvteam/igv#1538]
The left/right scroll arrows do not work.

![image](https://github.com/user-attachments/assets/f85d1e4c-79a7-445a-96e9-6e012fcbf548)

Error when refresh or navigating bam files
![image](https://github.com/user-attachments/assets/98c07470-5394-434f-ace6-94719f7f442f)
![image](https://github.com/user-attachments/assets/55d3d538-886b-46d1-b37f-d291878d7902)

**Comment by jrobinso** [comment]
Hi, could you clarify what you mean by "the left or right scroll arrows do not work"?    If you mean the arrows on the keyboard I cannot reproduce the problem.

Also, I cannot reproduce your BAM file problem.   There might be more information in your igv.log file,  located in the "igv" folder in your home directory.  

For both problems I will need detailed steps to reproduce.  If you cannot share your data you can try to reproduce the problems with the following,  use "File > Load from URL..."  for genome hg19

```
gs://genomics-public-data/platinum-genomes/bam/NA12877_S1.bam
```

**Comment by kevfengler227** [comment]
If you click on the arrows pointing left or right with mouse it does not pan left or right

**Comment by kevfengler227** [comment]
I see if I can pinpoint bam reloading issue...

**Comment by kevfengler227** [comment]
keyboard arrows work for panning, but not clicking on arrow

**Comment by jrobinso** [comment]
The arrow has never been used that way.

**Comment by kevfengler227** [comment]
in 2.17.3 it did, once "click and drag to zoom in" was engaged. This is how I navigate around so I noticed the difference immediately.

**Comment by jrobinso** [comment]
Really it didn't (the arrow that is).   Its not an active element its just drawn on a bitmap.   I'll try to reproduce what you observed in 2.17.3 but could you describe it more completely?  Exactly what steps you took.

**Comment by jrobinso** [comment]
Oh I see it.  That's actually a bug in 2.17.3.   Its not the arrow, clicking on the ruler should center where you clicked.   That does appear to be broken.

**Comment by jrobinso** [comment]
We'll fix it for the next point release.   BTW its not good practice to submit 2 bugs in one issue.   I'm going to rename this one for the click,  if you have a reproducible BAM problem (with steps I can take to reproduce) open a new issue.

**Comment by kevfengler227** [comment]
ah yes, it's not the arrow per se but the ruler. Clicking on the far left or far right of the ruler will advance the view by the same unit left or right in 2.17.3, which is the same as using arrows on keyboard.

will do.

**Comment by jrobinso** [comment]
This should be fixed in 2.18.1
